Dockerfile.template

```diff
@@ -29,7 +29,7 @@ EXPOSE 443
 ENV S6_LOGGING 0
 ENV S6_KEEP_ENV 1
 ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2
-ENV FTL_CMD debug
+ENV FTL_CMD no-daemon
```

Comment:
I'd say that this is actually the main change here

Reply:
Earlier comment was Part 1 main change

This is definitely part 2 main change.